xml/docu_styleguide.structure.xml

```diff
@@ -2785,9 +2786,11 @@ Entire hard disks are listed as devices without numbers, such as
     <para>
       Note that <tag class="emptytag">citetitle</tag> is not translated.
     </para>
-    <para>
+    <para outputformat="db">
```

Comment:
Adding `outputfromat="db"` for this paragraph makes it not visible in the adoc style guide. I guess the intention is different as the next sentence is profiled again for both formats.